test-network-function/accesscontrol/suite.go

```diff
@@ -130,16 +156,103 @@ func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase,
 	})
 }
 
+func getCrsNamespaces(crdName, crdKind string) map[string]string {
+	gomega.Expect(crdKind).NotTo(gomega.BeEmpty())
+	getCrNamespaceCommand := fmt.Sprintf(ocGetCrNamespaceFormat, crdKind)
+	cmdOut := utils.ExecuteCommand(getCrNamespaceCommand, common.DefaultTimeout, common.GetContext(), func() {
+		tcClaimLogPrintf("CRD %s: Failed to get CRs (kind=%s)", crdName, crdKind)
+	})
+	// No CRs created for this CRD yet.
+	if cmdOut == "" {
```

Comment:
looking at the whole code, this check is actually not needed. you can remove it and the same result will be returned